src/components/Option/Option.tsx

```diff
@@ -0,0 +1,7 @@
+interface Props {
+  option: number;
+}
+
+export const Option: React.FC<Props> = ({ option }) => {
+  return <option value={option}>{option}</option>;
```

Comment:
Ensure that the 'option' element has a 'key' prop if it's going to be used within a map function. This is a common React performance optimization to help with list reconciliation.